src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -142,6 +142,9 @@
         - name: --enable-vmss
           type: bool
           short-summary: (PREVIEW) Enable VMSS agent type.
+        - name: --node-zones
+          type: string array
+          short-summary: (PREVIEW) Availability zones where agent nodes will be placed.
```

Comment:
This will override the text you put in the help kwarg. You should remove this and, if desired, add "(PREVIEW)" to that help text.